js/client-teambuilder.js

```diff
@@ -463,11 +463,10 @@
 			}
 		},
 		defaultDragEnterTeam: function (e) {
-			// Drag originated outside teambuilder
-			if (!e.originalEvent.dataTransfer) return;
-			if (!e.originalEvent.dataTransfer.files) return;
-			if (e.originalEvent.dataTransfer.files[0] && e.originalEvent.dataTransfer.files[0].name.slice(-4) !== '.txt') return;
```

Comment:
You're not using the latest desktop client, since the latest desktop client doesn't support Windows XP.

Please use an OS that's remotely modern... XP doesn't get security updates, you're probably DDoSing PS as we speak...